PandasWorkshop.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>Generally this sort of looping is discouraged in pandas. I'm happy to help restructure this code so that it does not need a for loop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/orioncohen/metal-bands-by-nation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>